Bottom: 505e48e897dfa669e3516bd891cbea107d7238d9
Top:    3d95ffa4e600220fd21066e9e125f1059589c043
Author: Christopher Voltz <cjunk@voltz.ws>
Date:   2017-05-01 23:06:29 -0500

Refresh of rakefile

---

diff --git a/Rakefile b/Rakefile
index fa29060..7b4c20b 100644
--- a/Rakefile
+++ b/Rakefile
@@ -5,7 +5,5 @@ end
 
 desc "Reformat JS code"
 task :reformat_js do
-  mv "script.js", "script.js.input"
-  `uglifyjs --beautify --output script.js -- script.js.input`
-  rm "script.js.input"
+  `uglifyjs --beautify --output script.js -- script.js`
 end
